tests/phpunit/includes/UserTest.php

```diff
@@ -17,90 +17,14 @@ class UserTest extends MediaWikiTestCase {
 	public function setUp() {
 		parent::setUp();
 
-		$this->savedGroupPermissions = $GLOBALS['wgGroupPermissions'];
```

Comment:
What do you mean - they aren't used? So maybe we should just delete them, as otherwise other people like me will waste time on adjusting them.